tests.live/Python/test_live.py

```diff
@@ -79,7 +79,6 @@ def test_ionq_submit(self):
         """
         import time
         import qsharp
-        qsharp.projects.add(path.join(path.dirname(__file__), "qsharp", "ionq", "IonQ.csproj"))
```

Comment:
This is now injected via test fixtures, and should not be separately included here.